src/Compilers/CSharp/Test/Semantic/Diagnostics/DiagnosticAnalyzerTests.cs

```diff
@@ -715,8 +715,6 @@ public void TestDescriptorForConfigurableCompilerDiagnostics()
             var analyzer = new CSharpCompilerDiagnosticAnalyzer();
             foreach (var descriptor in analyzer.SupportedDiagnostics)
             {
-                Assert.True(descriptor.IsEnabledByDefault);
```

Comment:
Consider keeping the assertion, but listing the known exceptions to the rule

Reply:
I considered this, but the assert ended up feeling like: "assert that the behavior of IsEnabledByDefault on compiler diagnostics is the same as the behavior of IsEnabledByDefault on compiler diagnostics".